cmd/qat_plugin/qat_plugin.go

```diff
@@ -210,12 +211,17 @@ func (dp *devicePlugin) scan() (deviceplugin.DeviceTree, error) {
 	for _, driver := range append(dp.kernelVfDrivers, dp.dpdkDriver) {
 		files, err := ioutil.ReadDir(path.Join(dp.pciDriverDir, driver))
 		if err != nil {
-			return nil, fmt.Errorf("Can't read sysfs for driver %s: %+v", driver, err)
+			fmt.Printf("Driver %s not available: Skipping\n", driver)
+			continue
 		}
 
 		n := 0
 		for _, file := range files {
-			if !strings.HasPrefix(file.Name(), "0000:") {
+			vfdevID, err := dp.getDeviceID(file.Name())
+			if err != nil {
+				fmt.Printf("Cannot obtain deviceID for the device with PCI address: %s: %v", file.Name(), err)
+			}
```

Comment:
`getDeviceID()` failures manifest problems in the host like unreadable or disappearing files. Shouldn't we return an error here instead of ignoring the problems?